setup.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python
+
+"""
+Standard build script.
+"""
+
+from __future__ import print_function
+
+from os.path import dirname, join
+
+try:
+    from setuptools import setup, find_packages
+except ImportError:
+    print("You must have setuptools installed to use setup.py. Exiting...")
+    raise SystemExit(1)
+
+__docformat__ = 'restructuredtext'
+
+# Import requirements
+with open(join(dirname(__file__), 'requirements.txt')) as f:
+    required = f.read().splitlines()
+
+setup(
+    name="python-owasp-zap-v2.4",
```

Comment:
To do that, I think a better approach with having only one library, but with a different API. This is:

**For version 2.4 of ZAP**
`from zap import ZAP_24`
`c = ZAP_24()`

**For version 2.5 of ZAP**
`from zap import ZAP_25`
`c = ZAP_25()`

And so on. This way, all ZAP versions are unified in only one library.